klippa/src/parsing_util.rs

```diff
@@ -86,6 +87,48 @@ pub fn parse_drop_tables(input_str: &str) -> Result<IntSet<Tag>, SubsetError> {
     Ok(result)
 }
 
+//parse input name_IDs string, which is a comma/whitespace-separated list of nameIDs that will be retained
+pub fn parse_name_ids(input_str: &str) -> Result<IntSet<NameId>, SubsetError> {
+    let mut result = IntSet::empty();
+
+    if input_str == "*" {
+        let out = IntSet::<NameId>::all();
+        return Ok(out);
```

Comment:
I wouldn't have anticipated needing a binding; does `return Ok(IntSet::all())` not infer the type?